resolver/resolver.go

```diff
@@ -151,7 +151,20 @@ func (a Address) Equal(o Address) bool {
 
 // String returns JSON formatted string representation of the address.
 func (a Address) String() string {
-	return pretty.ToJSON(a)
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("{Addr: %v, ", a.Addr))
+	sb.WriteString(fmt.Sprintf("ServerName: %v, ", a.ServerName))
+	var atrStr string
+	if a.Attributes != nil {
+		atrStr = a.Attributes.String()
+	}
+	sb.WriteString(fmt.Sprintf("Attributes: %v, ", atrStr))
```

Comment:
I think it make more sense to do this only if `a.Attributes != nil`.

Reply:
Good point. Switched.